simapp/app.go

```diff
@@ -313,7 +313,7 @@ func NewSimApp(
 	app.CrisisKeeper = crisiskeeper.NewKeeper(appCodec, keys[crisistypes.StoreKey], invCheckPeriod,
 		app.BankKeeper, authtypes.FeeCollectorName, authtypes.NewModuleAddress(govtypes.ModuleName).String())
 
-	app.FeeGrantKeeper = feegrantkeeper.NewKeeper(appCodec, keys[feegrant.StoreKey], app.AccountKeeper)
+	app.FeeGrantKeeper = feegrantkeeper.NewKeeper(appCodec, keys[feegrant.StoreKey], app.AccountKeeper, app.AccountKeeper.GetAddressCodec())
```

Comment:
app.AccountKeeper has already the two methods `AddressStringToBytes` and `AddressBytesToString`. I'm not sure it's worth passing the address.Codec to FeegrantKeeper.